scripts/zones/Bearclaw_Pinnacle/mobs/Eldertaur.lua

```diff
@@ -0,0 +1,20 @@
+-----------------------------------
+-- Area: Bearclaw Pinnacle
+--  Mob: Eldertaur
+--  ENM: Brothers
+-----------------------------------
+require("scripts/globals/status")
+-----------------------------------
+
+function onMobInitialize(mob)
+    mob:setMobMod(dsp.mobMod.DRAW_IN, 1)
+    mob:setMod(dsp.mod.DMGMAGIC, -10)
+    mob:setMod(dsp.mod.SLEEPRES, 75)
+end
+
+function onMobSpawn(mob)
+    mob:setMobMod(dsp.mobMod.SUPERLINK, mob:getShortID())
```

Comment:
@teschnei didn't we have this being handled automatically for bcnm mobs without having to set this?

Reply:
We should fix that if it broke

Reply:
Probably because of https://github.com/DarkstarProject/darkstar/issues/6119

Reply:
yeah ok, we needa fix that then after we'll nuke all these manual superlinks. waiting on appveyor atm to merge this.